src/modules/loadPage.ts

```diff
@@ -41,16 +41,16 @@ export function performPageLoad(this: Swup, data: PageLoadOptions) {
 	// start/skip animation
 	const animationPromises = this.leavePage({ event, skipTransition });
 
+	// Load page data
+	const fetchPromise = this.fetchPage(data);
+
 	// create history record if this is not a popstate call (with or without anchor)
 	if (!isHistoryVisit) {
 		createHistoryRecord(url + (this.scrollToElement || ''));
 	}
```

Comment:
It's not related to this PR, and I think the behaviour still works fine since it's a sync code even with he order shuffled in this file (after previous refactoring), but just something to keep in mind... some browsers (like Safari on iPhone) create screenshot of the page which is visible when navigating back. This screenshot is created at the moment of changing the URL with `pushState`, so doing it in the process of transition might lead to page being faded away or similar on the screenshot.

But again, just noting the use case, all good here.